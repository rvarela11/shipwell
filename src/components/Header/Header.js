// @vendors
import React from 'react';

// @styles
import './Header.scss';

const Header = () => (
    <header className="header">
        <div className="header__logo" />
    </header>
);

export default Header;
