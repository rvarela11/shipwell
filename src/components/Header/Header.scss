// @styles
@import '../../globals/scss/colors';
@import '../../globals/scss/spacing';
@import '../../globals/scss/typography';

.header {
    padding: $spacing-md;
}

.header__logo {
    background-image: url(../../images/shipwell-logo.png);
    background-repeat: no-repeat;
    background-size: contain;
    background-position: center;
    flex-grow: 1;
    height: rem(60px);
    margin: 0;
    padding: 0;
}
