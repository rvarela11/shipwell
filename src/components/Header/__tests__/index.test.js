// @vendors
import React from 'react';
import { shallow } from 'enzyme';

// @components
import Header from '../Header';

describe('Header', () => {
    it('renders Header without crashing', () => {
        const wrapper = shallow(<Header />);
        expect(wrapper).toMatchSnapshot();
    });
});
