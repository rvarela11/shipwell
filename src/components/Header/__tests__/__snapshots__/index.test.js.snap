// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders Header without crashing 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Header />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        className="header__logo"
      />,
      "className": "header",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "className": "header__logo",
      },
      "ref": null,
      "rendered": null,
      "type": "div",
    },
    "type": "header",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="header__logo"
        />,
        "className": "header",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "className": "header__logo",
        },
        "ref": null,
        "rendered": null,
        "type": "div",
      },
      "type": "header",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
